src/userGuide/UserGuideImage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import styles from './UserGuide.module.css';
+
+interface ImageProps {
+  alt: string;
+  src: string;
+}
+
+function UserGuideImage({ src, alt }: ImageProps): React.ReactElement {
+  return (
+    <p>
+      <img src={src} alt={alt} />
```

Comment:
Could try adding `loading="lazy"`, could be nice since there are so many pictures to be loaded 🤔

Reply:
Everything is lazy now